[PLT-1652] Allow skipping kubernetes intermediate version during upgrade (#718)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/upgrade.adoc

|===
| Flag | Description | Default Value | Mandatory

| `-p`, `--vault-password`

> | File containing the Vault password needed to decrypt secrets.

Reviewer: ```suggestion
| Specifies a file with the Vault password to decrypt secrets.
```

> |

Reviewer: ```suggestion
| None
```

| Disable environment preparation for the upgrade process.
| False
| No

| `--upgrade-cloud-provisioner-only`

> | Prepare the upgrade process for the cloud-provisioner upgrade only. This is only supported in EKS environments.

Reviewer: ```suggestion
| Updates only the _cloud-provisioner_ components. Compatible only with EKS environments.
```

> | False

Reviewer: ```suggestion
| False
| No
```

| `--skip-k8s-intermediate-version`

> | Skips updating the workers to the intermediate Kubernetes version. This is only supported in EKS environments.

Reviewer: ```suggestion
| Skips the upgrade of _workers_ to an intermediate Kubernetes version. Compatible only with EKS environments.
```
